### DISCUSSION TITLE
addon ??

### DISCUSSION THREAD

**Discussion by pressurejets** [skrashevich/double-take#292] [General]
https://github.com/skrashevich/double-take is not a valid add-on repository

its this version not a repository addon ?? 
which version is being maintained ??

**Comment by pressurejets** [comment]
> https://github.com/skrashevich/double-take is not a valid add-on repository
> 
> its this version not a repository addon ?? which version is being maintained ??

fixed had to refresh sorry

**Comment by skrashevich** [comment]
add-on repo: https://github.com/skrashevich/hassio-addons
